test/util/src/main/java/org/glassfish/fighterfish/test/util/ZipUtil.java

```diff
@@ -99,8 +99,10 @@ public static void extractZip(final ZipInputStream zis, final File destDir)
             LOGGER.logp(Level.FINER, "ZipUtil", "extractZip",
                     "ZipEntry name = {0}, size = {1}",
                     new Object[]{ze.getName(), ze.getSize()});
-            java.io.File f = new java.io.File(destDir + java.io.File.separator
-                    + ze.getName());
+            java.io.File f = new File(destDir, ze.getName());
+            if (!f.toPath().normalize().startsWith(destDir.toPath().normalize())) {
```

Comment:
Currently this fails checkstyle:
```
12:28:41  [INFO] There is 1 error reported by Checkstyle 8.18 with /home/jenkins/agent/workspace/fighterfish_PR-46/checkstyle.xml ruleset.
12:28:41  [ERROR] src/main/java/org/glassfish/fighterfish/test/util/ZipUtil.java:[103] (sizes) LineLength: Line is longer than 80 characters (found 83).
```

Reply:
Does this need to be changed if you intend to increase the line length to 120?

Reply:
If 120 is accepted then - no, it doesn't.